lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -35,6 +35,14 @@ module Vmpooler
           'platform' => 'el-7-x86_64',
           'template' => 'centos-7-x86_64'
         },
+        'cisco5-64' => {
+          'platform' => 'cisconxos-9-x86_64',
+          'template' => 'cisco-nxos-9k-x86_64'
```

Comment:
Since the nxos template isn't on vmpooler yet, I'd prefer to postpone adding it here.